src/wasm-ops/clip.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "common.h"
+
+extern "C" {
+void clip_f32(void *);
+// Expand for other supported data types for `clip`
+}
+
+// Core implementation of the op
+template <typename T>
+void clip_imp(const T *input, T *output, const int32_t length, const float min,
+              const float max) {
+  for (size_t i = 0; i < length; ++i) {
+    const auto &val = input[i];
+    output[i] = (val < min) ? min : (val > max) ? val : val;
```

Comment:
output[i] = (val < min) ? min : (val > max) ? **`max`** : val;

Reply:
Thanks for catching this. Will fix.

Reply:
This is weird, I am not sure, how this is not bieng caught by node tests. I tried addign unit tests, but I realized that our test data sits in some toher repo making it hard to add tests :(

Reply:
op test is in the repo and node tests are not. do you want to add test case into node tests?

Reply:
I had not resolved this op in the `wasm` backend and so it was passing (despite the bug) all tests by falling back to the  `cpu` backend. I added `wasm` native clip in the operators resolve function and this caught the bug (as expected)